server/rootless.go

```diff
@@ -45,3 +83,27 @@ func makeOCIConfigurationRootless(g *generate.Generator) {
 
 	g.SetLinuxCgroupsPath("")
 }
+
+// getAvailableV2Controllers returns the entries in /sys/fs/cgroup/<SELF>/cgroup.controllers
+func getAvailableV2Controllers() map[string]struct{} {
+	procSelfCgroup, err := cgroups.ParseCgroupFile("/proc/self/cgroup")
+	if err != nil {
+		logrus.WithError(err).Error("failed to parse /proc/self/cgroup")
```

Comment:
nit: I think the error message already contains file name, so wrapping is redundant. Not a big deal though.

Reply:
updated